### ISSUE TITLE
Cannot download certain movies if they have same seasonId and episodeId

### ISSUE THREAD

**Issue by Bommel24** [crunchy-labs/crunchy-cli#199]
**Describe the bug**
Can't download for example 1st movie of Fairy Tail Movies because it always downloads the second one.

**To Reproduce**
Try to archive the fairy tail movies
```
$ crunchy archive -o test.mkv https://www.crunchyroll.com/series/G4PH0WMN7/fairy-tail-movies
```

**Expected behavior**
Download both movies or let them be selectable via id eg [S2E0]

**Client (please complete the following information):**
 - OS: Windows 11
 - Version crunchy-v3.0.0-dev.13_windows

**Additional context**
Problem is that both movies are seasonId 1 and episodeId null maybe make them selectable via identifier [G4PH0WMN7|M1|E0] or [G4PH0WMN7|M2|E0] or via any other method.

**Comment by ByteDream** [comment]
I though Crunchyroll fixed all season numbering... well. As a fix you can pass the url of each movie to `crunchy-cli`. I have to see how I get around this issue.
E.g.
```
crunchy archive -o test.mkv https://www.crunchyroll.com/watch/G2XU03VG5/dragon-cry-german-dub https://www.crunchyroll.com/watch/GZ7UVXMEJ/
```
(the urls may differ for you, depending on your country)

**Comment by ByteDream** [comment]
Interactive choosing of season which duplicated numbers is now (re-)implemented in [v3.0.0-dev.15](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.0.0-dev.15).
